### PR TITLE
fix plugin slice throttle not work

### PR DESCRIPTION
**Pull request by zds05**

### CODE REVIEW

File: iocore/aio/AIO.cc [hunk 1]
```diff
@@ -24,7 +24,6 @@
 /*
  * Async Disk IO operations.
  */
-
```
Comment (randall) [inline iocore/aio/AIO.cc:27]: can you remove this line removal? It doesn't look related to the slice plugin

Comment (traeak) [reply iocore/aio/AIO.cc:27]: agreed.  This PR should fix a dumb bug.
